packages/react-devtools-inline/__tests__/__e2e__/inspecting-props.test.js

```diff
@@ -4,49 +4,131 @@ const {test, expect} = require('@playwright/test');
 const config = require('../../playwright.config');
 test.use(config);
 
-test.describe('Testing Todo-List App', () => {
-  let page, frameElementHandle, frame;
-  test.beforeAll(async ({browser}) => {
+test.describe('ListApp', () => {
+  let page;
+
+  test.beforeEach(async ({browser}) => {
```

Comment:
`beforeEach` is better than `beforeAll` because it resets the test app state between tests (reducing likelihood of fragile coupling between tests).

Reply:
yes, i agree

Reply:
`beforeAll` is good for apps where state and site data are not changing too much. It slows down tests a little bit. But `beforeEach` is also good it ensures that whatever the data and state are fresh.